digital-thoreau-simplify/usr/share/asteroid-launcher/watchfaces/digital-thoreau-simplify.qml

```diff
@@ -0,0 +1,132 @@
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.12
+import QtGraphicalEffects 1.12
+
+Item {
+
+    Image {
+        z: 1
+        id: asteroidLogo
+        visible: !displayAmbient
+        source: "../watchfaces-img/asteroid-logo.svg"
+        antialiasing: true
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: -parent.height * 0.272
+        }
+        width: parent.width * 0.12
+        height: parent.height * 0.12
+        opacity: 0.7
+
+        Text {
+            z: 2
+            id: asteroidSlogan
+            visible: !displayAmbient
+            font.pixelSize: parent.height * 0.31
+            font.family: "Raleway"
+            color: "white"
+            horizontalAlignment: Text.AlignHCenter
+            anchors {
+                centerIn: parent
+                verticalCenterOffset: -parent.height * 0.005
+            }
+            text: "<b>AsteroidOS</b><br>Hack Your Wrist"
+        }
+        MouseArea {
+            anchors.fill: parent
+            onPressAndHold: asteroidLogo.visible ?
+                                asteroidLogo.visible = false :
+                                asteroidLogo.visible = true
+        }
+    }
+
+    Text {
+        id: datetext
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: parent.height * 0.25 
+        }
+        font.family: "Titillium"
+        font.weight: Font.Thin
+        font.pixelSize: parent.height * 0.06
+        color: "white"
+        text: wallClock.time.toLocaleString(Qt.locale(), "ddd dd");
+    }
+
+    Text {
+        id: time
+        anchors.centerIn: parent
+        font.family: "Titillium"
+        font.weight: Font.Thin
+        font.pixelSize: parent.height * 0.30
+        color: "white"
+        text: wallClock.time.toLocaleString(Qt.locale(), (use12H.value ? "hhmm ap" : "HHmm")).slice(0,4);
+    }
+
+    Text {
+        id: ampm
+        visible: use12H.value
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: parent.height * 0.143
+        }
+        font.family: "Titillium"
+        font.weight: Font.Bold
+        font.pixelSize: parent.height * 0.05
+        color: "white"
+        text: wallClock.time.toLocaleString(Qt.locale(), "ap");
+    }
+
+    Repeater{
+        id: secondMarks
+        model: 60
+        Rectangle {
+            id: second
+            visible: !displayAmbient
+            antialiasing : true
+            color: "transparent"
+            width: parent.width * 0.01
+            height: parent.height * 0.125
+            transform: [
+                Rotation { 
+                    origin.x: second.width/2
+                    origin.y: parent.height * 0.48
+                    angle: (index)*360/secondMarks.count 
+                },
+                Translate { 
+                    x: (parent.width - second.width)/2
+                    y: parent.height/2 - parent.height * 0.48
```

Comment:
There are spacings around the ` * ` but non around `/`. As a code reading beginner, i feel it is much faster to grasp equasion like code when the operators have spacings. Also, a tiny bit more beautiful for the consistency nerd in us all imo ;)

Reply:
Good point -- fixed.